MapboxCoreNavigation/RouteController.swift

```diff
@@ -676,6 +685,30 @@ extension RouteController: CLLocationManagerDelegate {
         }
     }
     
+    func monitorIntersectionProgress(_ location: CLLocation) {
```

Comment:
“Monitor” isn’t a good word to use for this method (or `monitorStepProgress(_:)` for that matter), since it doesn’t set up its own location manager or anything. Additionally, it updates the step progress just like `monitorStepProgress(_:)` does; there isn’t a separate IntersectionProgress class to update.

How about `updateDistanceToIntersection(from:)`?